.gitlab-ci.yml

```diff
@@ -46,7 +46,7 @@ workflow:
     - rustc -vV
     - rustup show
     - bash --version
-    - ./scripts/pre_cache.sh
+    - ./scripts/.ci/pre_cache.sh
```

Comment:
why hidden?

Reply:
IMHO it's better practice to have stuff that's not part of the core-ink! project itself be hidden (like the stuff in `.github/`). The scripts in this `scripts` folder are meant to be executed by developers contributing to ink!, I feel that having unrelated CI stuff visibly in there is confusing.

Reply:
ok whatever, I'm going to standardize it a bit later